tests/configs/k8s_cronjobs_beta_api_version/k8s_cronjobs_beta.yml

```diff
@@ -82,7 +83,16 @@ spec:
             image: generic-image
             imagePullPolicy: 'Always'
             name: generic-container
+            volumeMounts:
+            - {name: secret-volume, mountPath: /mnt/cronjobs-secrets-store, readonly: True}
           restartPolicy: OnFailure
+          volumes:
+          - name: secret-volume
+            csi:
+              driver: secrets-store.csi.k8s.io
+              readOnly: true
+              volumeAttributes:
+                secretProviderClass: aws-secrets
```

Comment:
To @gchazot 's point in standup, can we remove these tests from the original files and instead add secret specific test files? These tests are specific to a piece of functionality (cronjobs) so we should create a testfile specific to secrets.

The secret testfile could have a `dependancies` `app` and `cronjob` all in one file with different mounted secretfiles.

Reply:
The reason is we want the tests to run against these files when secret's _are not_ mounted, so we know they won't break without secrets.

Reply:
Reverted the tests in this commit: https://github.com/EDITD/kubetools/pull/153/commits/90fbd0de0993d5d2bcbb2ff94e906367b478820b

Added new tests in this commit: https://github.com/EDITD/kubetools/pull/153/commits/e898c4fdb81bdc5bf0376f5c04d2f6ddfa9bfa31